content/docs/tasks/security/vault-ca/index.md

```diff
@@ -69,53 +65,73 @@ env:
 
     {{< text bash >}}
-    $ kubectl apply -f httpbin-injected.yaml
+    $ kubectl create serviceaccount vault-citadel-sa
+    {{< /text >}}
+
+1.  Since the Vault CA requires the authentication and authorization of Kubernetes service accounts,
+    you must edit the `vault-citadel-sa` service account to use the example JWT configured
+    on the testing Vault CA.
+    To learn more about configuring a Vault CA for Kubernetes authentication and authorization,
+    visit the [Vault Kubernetes `auth` method reference documentation](https://www.vaultproject.io/docs/auth/kubernetes.html).
+
+    {{< text bash >}}
+    $ export SA_SECRET_NAME=$(kubectl get serviceaccount vault-citadel-sa -o=jsonpath='{.secrets[0].name}')
+    $ kubectl edit secret ${SA_SECRET_NAME}
+
+    # When editing the secret, change the field "token" to: ZXlKaGJHY2lPaUpTVXpJMU5pSXNJbXRwWkNJNklpSjkuZXlKcGMzTWlPaUpyZFdKbGNtNWxkR1Z6TDNObGNuWnBZMlZoWTJOdmRXNTBJaXdpYTNWaVpYSnVaWFJsY3k1cGJ5OXpaWEoyYVdObFlXTmpiM1Z1ZEM5dVlXMWxjM0JoWTJVaU9pSmtaV1poZFd4MElpd2lhM1ZpWlhKdVpYUmxjeTVwYnk5elpYSjJhV05sWVdOamIzVnVkQzl6WldOeVpYUXVibUZ0WlNJNkluWmhkV3gwTFdOcGRHRmtaV3d0YzJFdGRHOXJaVzR0Y21aeFpHb2lMQ0pyZFdKbGNtNWxkR1Z6TG1sdkwzTmxjblpwWTJWaFkyTnZkVzUwTDNObGNuWnBZMlV0WVdOamIzVnVkQzV1WVcxbElqb2lkbUYxYkhRdFkybDBZV1JsYkMxellTSXNJbXQxWW1WeWJtVjBaWE11YVc4dmMyVnlkbWxqWldGalkyOTFiblF2YzJWeWRtbGpaUzFoWTJOdmRXNTBMblZwWkNJNklqSXpPVGs1WXpZMUxUQTRaak10TVRGbE9TMWhZekF6TFRReU1ERXdZVGhoTURBM09TSXNJbk4xWWlJNkluTjVjM1JsYlRwelpYSjJhV05sWVdOamIzVnVkRHBrWldaaGRXeDBPblpoZFd4MExXTnBkR0ZrWld3dGMyRWlmUS5STkgxUWJhcEpLUG1rdFYzdENucGl6N2hvWXB2MVRNNkxYelRoT3RhRHA3TEZwZUFOWmNKMXpWUWR5czNFZG5sa3J5a0dNZXBFanNkTnVUNm5kSGZoOGpSSkFadU5XTlBHcmh4ejRCZVVhT3FaZzN2N0F6SmxNZUZLallfZmlUWVlkMmdCWlp4a3B2MUZ2QVBpaEhZbmcyTmVOMm5LYmlaYnNuWk5VMXFGZHZiZ0NJU2FGcVRmMGRoNzVPemdDWF8xRmg2SE9BN0FOZjdwNTIyUERXX0JSbG4wUlR3VUpvdkNwR2VpTkNHZHVqR2lOTERaeUJjZHRpa1k1cnlfS1hUZHJWQWNUVXZJNmx4d1JiT05OZnVOOGhySURsOTV2SmpoVWxFLU8tX2N4OHFXdFhOZHFKbE1qZTFTc2lQQ0w0dXE3ME9lcEdfSTRhU3pDMm84YUR0bFE=
     {{< /text >}}
 
-1.  List Node Agent's pods:
+1.  Deploy the `httpbin` and `sleep` backends:
 
     {{< text bash >}}
-    $ kubectl get pod -n istio-system -l app=nodeagent -o jsonpath={.items..metadata.name}
+    $ kubectl apply -f httpbin-injected.yaml
+    $ kubectl apply -f sleep-injected.yaml
     {{< /text >}}
 
-1.  View each Node Agent's logs. The Node Agent residing on
-the same node as the testing workload will contain Vault related logs.
+## Istio mutual TLS with Vault CA integration
+
+This section demos the use of mutual TLS with the Vault CA integration. In the previous steps,
+you enabled mutual TLS for the Istio mesh and the `httpbin` and `sleep` workloads. These workloads receive
+certificates from a testing Vault CA. When you send a `curl` request from the `sleep` workload
+to the `httpbin` workload, the request goes through a mutual TLS protected channel constructed
+with the certificates the Vault CA issued.
 
     {{< text bash >}}
```

Comment:
This code block is not rendering with bash syntax highlighting. @geeknoid do you know why?

Reply:
lines 114-118 should come after line 122

Reply:
Done: 1. changed lines 114-118 to come after line 122. 2. rebased to fix the syntax highlighting.